.github/workflows/check_setup_py.yaml

```diff
@@ -27,3 +27,18 @@ jobs:
     - name: Check if something changed after running the update script
       run: |
         [ -z "$(git status --porcelain=v1 -uno 2>/dev/null)" ] 
+  check-version-numbers:
```

Comment:
If we add the check here, we should rename the workflow

Reply:
I will move it to a new workflow